rpyc/rpyc_slave/slave_server.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import os
+
+sys.path.append(os.getcwd())
+
+from rpyc.utils.server import ThreadedServer
+from rpyc_slave.FileMonitorService import FileMonitorService
+
+"""
+Executes the slave server.
+Run this as (from the rpyc project's root directory):
```

Comment:
For some reason, it is. It otherwise it says ModuleNotFoundError or ImportError.
You have to run it from the rpyc directory using "python -m rpyc_slave.slave_server".

Reply:
correct. Fixed it now